ui/desktop/electron-app/package.json

```diff
@@ -41,8 +41,7 @@
   },
   "resolutions": {
     "highlight.js": "^10.4.1",
-    "jsonpointer": "^5.0.0",
-    "node-fetch": "^2.6.7",
```

Comment:
Doesn't have a change within `yarn.lock` because it did not change the current resolution of `node-fetch`.